src/constants/numerical.js

```diff
@@ -47,5 +47,11 @@ module.exports = {
     /*
      * Are two values nearly equal? Compare to 1PPM
      */
-    ALMOST_EQUAL: 1 - 1e-6
+    ALMOST_EQUAL: 1 - 1e-6,
+
+    /*
+     * not a number, but for displaying numbers: the "minus sign" symbol is
+     * wider than the regular ascii dash "-"
+     */
+    MINUS_SIGN: '\u2212'
```

Comment:
Currently this is only used in cartesian tick labels (and hover text) and this does get reused in other places... but I suspect there are places in other plot types where we use a regular dash that should be replaced by this one.